src/utils/__tests__/parseJsDoc-test.js

```diff
@@ -64,6 +64,21 @@ describe('parseJsDoc', () => {
       });
     });
 
+    it('extracts jsdoc union type param', () => {
+      const docblock = `
+        @param {string|Object} bar
+      `;
+      expect(parseJsDoc(docblock)).toEqual({
+        description: null,
+        returns: null,
+        params: [{
+          name: 'bar',
+          type: {name: 'string|Object'},
```

Comment:
I'm wondering if this shouldn't rather be something like `{name: 'union', value: ['string', 'Object']}`. What do you think @janicduplessis?